.github/workflows/build-nightly-images.yaml

```
@@ -88,16 +110,22 @@ jobs:
       matrix:
         image: ${{ fromJSON(inputs.matrix-config).image }}
     steps:
-      - name: Get Nightly Tag
+      - name: Get Tag
         id: get_tag
         shell: bash
+        env:
+          INPUT_TAG: ${{ inputs.tag }}
         run: |
```

actionlint (1.7.11)

[error] shellcheck reported issue in this script: SC2086:info:11:39: Double quote to prevent globbing and word splitting

(shellcheck)